src/js/Keyboard.js

```diff
@@ -152,6 +147,24 @@ class Keyboard extends Component {
     return keyboardLayout;
   }
 
+  onMouseDown = e => {
+    this.setState({ ...this.state, isScrolling: true, 
+     clientX: e.clientX });
+  };
+
+  onMouseUp = () => {
+    this.setState({ ...this.state, isScrolling: false, clientX: 0, scrollX: 0 });
+  };
+
+  onMouseMove = e => {
+    const { clientX, scrollX } = this.state;
+    if (this.state.isScrolling) {
```

Comment:
Note that if you release the mouse while not hovering over the autocomplete list, the scroll doesn't terminate properly

Reply:
Addressed